src/utils/promptRunner.ts

```diff
@@ -0,0 +1,107 @@
+import { spawn } from "child_process";
+import { window } from "vscode";
+const output = window.createOutputChannel("Docker Labs: AI Tools");
+
+export const getRunArgs = (promptRef: string, projectDir: string, username: string, platform: string, pat: string, render = false) => {
+    const isLocal = promptRef.startsWith('local://');
+    let promptArgs: string[] = ["--prompts", promptRef];
+    let mountArgs: string[] = [];
+
+    if (isLocal) {
+        const localPromptPath = promptRef.replace('local://', '');
+        const pathSeparator = platform === 'win32' ? '\\' : '/';
+        promptRef = localPromptPath.split(pathSeparator).pop() || 'unknown-local-prompt';
+        promptArgs = ["--prompts-dir", `/app/${promptRef}`];
+        mountArgs = ["--mount", `type=bind,source=${localPromptPath},target=/app/${promptRef}`];
+    }
+
+    const baseArgs: string[] = [
+        'run',
+        '--rm',
+        '-v', '/var/run/docker.sock:/var/run/docker.sock',
+        '-v', 'openai_key:/root',
```

Comment:
I don't think /root is a good mount point name since it's so common for images to use /root as a $HOME directory for the root account.  It'd be better if we could name it something like /secret but I can see why you have have had to do this (because /root is the $HOME and we're searching for $HOME/.openai-api-key.  Should we just use an environment variable to point at where the openai key will be written?  (eg OPENAI_API_KEY_LOCATION).  That way you can set it to something like /secret/.openai-api-key and not risk weird bugs if we ever change the Docker image to use a non-root user.

Reply:
That feels much more flexible. I thought since the only images here are our own, using root was safe.